debug-bar-searchpress.php

```diff
@@ -110,7 +110,8 @@ public function sp_rp_args( $args ) {
 	}
 
 	public function post_request( $response, $type, $class, $args, $url ) {
-		if ( preg_match( "#/post/_search#i", $url ) ) {
+		// Account for ES 6.0 and 7.0 doc types.
+		if ( preg_match( "#/post/_search#i", $url ) || preg_match( "#/_doc/_search#i", $url ) ) {
```

Comment:
```suggestion
		if ( preg_match( "#/(post|_doc)/_search#i", $url ) ) {
```

Reply:
:face-palm: how did that go over my head haha